client/src/plugins/camunda-plugin/deployment-tool/DeploymentTool.js

```diff
@@ -485,3 +485,11 @@ function addOrUpdateById(collection, element) {
     element
   ];
 }
+
+function isCamundaTab(tab) {
```

Comment:
Will we not require the usage of this function also in other plugins (e.g., StartInstanceTool)? If so, maybe we could put it into CamundaPlugin.js and import from there. The location would then be consistent to https://github.com/camunda/camunda-modeler/blob/make-deployment-camunda-exclusive/client/src/plugins/zeebe-plugin/ZeebePlugin.js#L88. Also note that we call them differently `isCamundaTab()...` vs `isZeebeDiagram()...`.

Reply:
> Will we not require the usage of this function also in other plugins (e.g., StartInstanceTool)?

Not really, since StartInstanceTool will only work for bpmn tabs, not dmn tabs nor cmmn tabs.

One note regarding https://github.com/camunda/camunda-modeler/blob/make-deployment-camunda-exclusive/client/src/plugins/zeebe-plugin/ZeebePlugin.js#L88

It's basically not working properly, because we use the same subscriber on the base plugin, and in its children, which follows to crashed Modelers due to endless loops. We will fix it with https://github.com/camunda/camunda-modeler/commit/9d442af36fd9dcb13cc8d7a0312055afb54aa7cb in a different PR.

I agree with the naming convention, but I wouldn't place the function inside CamundaPlugin to avoid circled imports. Personally, I can live with the code duplication in this place.

Reply:
:+1: Then lets go with the renaming and merge

Reply:
So `isCamundaTab` is okay, right? It's more explicit since we're dealing with tabs, not diagrams :).

 I'd re-phrase it for Zeebe in a different place ([`9d442af`](https://github.com/camunda/camunda-modeler/commit/9d442af36fd9dcb13cc8d7a0312055afb54aa7cb#diff-5f32eb059df8b990c64b69db90c7e7fa2c0b34755673837a05b8491c8cdb1475R425)).